Example/iOS Example/AppDelegate.swift

```diff
@@ -43,6 +43,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 return
         }
         PiwikTracker.configureSharedInstance(withSiteID: siteid, baseURL: baseurl)
+        PiwikTracker.shared?.logger?.minLevel = .info
```

Comment:
Maybe look at Alamofire if you want to offer a "default" instance without having to instantiate one.